wp-content/themes/citylimits/tribe-events/single-event.php

```diff
@@ -3,86 +3,88 @@
  * Single Event Template
  * A single event. This displays the event title, description, meta, and
  * optionally, the Google map for the event.
- * 
+ *
  * Override this template in your own theme by creating a file at [your-theme]/tribe-events/single-event.php
  *
  * @package TribeEventsCalendar
+ * @version 4.6.19
  *
  */
 
-if ( !defined('ABSPATH') ) { die('-1'); }
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
```

Comment:
You may wish to use https://codex.wordpress.org/Function_Reference/wp_die

Reply:
This is "tribe-events" code, not mine, but fixed anyhow.